apps/provider-scaffold/template/config/config.go.template

```diff
@@ -0,0 +1,18 @@
+package config
+
+import "go.mondoo.com/cnquery/providers-sdk/v1/plugin"
+
+var Config = plugin.Provider{
+	Name:    "{{ .ProviderID }}",
+	ID:      "{{ .GoPackage }}",
+	Version: "9.0.0",
```

Comment:
Only a suggestion, as we use this in some of our providers.

Add:
```
MinArgs: 1,
MaxArgs: 1,
```

Reply:
If we add this, this provider would not work after compile since we need an argument.